server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/organization/MembersPage.java

```diff
@@ -18,54 +18,50 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-package org.sonarqube.pageobjects.organization;
+package org.sonarqube.qa.util.pageobjects.organization;
 
 import com.codeborne.selenide.Condition;
 import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.Selenide;
 import com.codeborne.selenide.SelenideElement;
 
-import static com.codeborne.selenide.Condition.text;
-import static com.codeborne.selenide.Condition.visible;
-import static com.codeborne.selenide.Selenide.$;
-import static com.codeborne.selenide.Selenide.$$;
-
 public class MembersPage {
 
   public MembersPage() {
-    $(".navbar-tabs a.active").shouldBe(visible).shouldHave(text("Members"));
+    Selenide.$(".navbar-tabs a.active").shouldBe(Condition.visible).shouldHave(Condition.text("Members"));
   }
 
   public ElementsCollection getMembers() {
-    return $$("table.data tr");
+    return Selenide.$$("table.data tr");
   }
 
   public MemberItem getMembersByIdx(Integer idx) {
     return new MemberItem(getMembers().get(idx));
   }
 
   public MembersPage shouldHaveTotal(int total) {
-    $(".panel-vertical > span > strong").shouldHave(text(String.valueOf(total)));
+    Selenide.$(".panel-vertical > span > strong").shouldHave(Condition.text(String.valueOf(total)));
     return this;
   }
 
   public MembersPage searchForMember(String query) {
-    $(".page .search-box-input").shouldBe(visible).val("").sendKeys(query);
+    Selenide.$(".page .search-box-input").shouldBe(Condition.visible).val("").sendKeys(query);
     return this;
   }
 
   public MembersPage canAddMember() {
-    $(".page-actions").shouldBe(visible);
+    Selenide.$(".page-actions").shouldBe(Condition.visible);
     return this;
   }
 
   public MembersPage canNotAddMember() {
-    $(".page-actions").shouldNot(Condition.exist);
+    Selenide.$(".page-actions").shouldNot(Condition.exist);
     return this;
   }
 
   public MembersPage addMember(String login) {
     this.canAddMember();
-    $(".page-actions button").click();
+    Selenide.$(".page-actions button").click();
 
     SelenideElement modal = this.getModal("Add user");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2209)